src/manifests_workflow/component_opensearch.py

```diff
@@ -21,6 +21,7 @@ def checkout(
         name: str,
         path: str,
         opensearch_version: str,
+        repo_url: str,
```

Comment:
Shall we also update the `GitRepository` below? This parameter seems not be used and picked up from the `GitRepository` initialization.

Reply:
This has not been removed to preserve the old interface to make sure other tests are not affected by this change. @zelinh

Reply:
Had an offline discussion with @rishabh6788 so there was a miss and updated in the latest commit below.